src/dashboard/src/app/client/admin/users/page.tsx

```diff
@@ -49,6 +50,12 @@ export default function Page() {
   const [isSubmitting, setIsSubmitting] = React.useState(false);
   const [organization, setOrganization] = React.useState('');
 
+  const isDirty = formUsers.some((u) => u.isDirty);
```

Comment:
Regrettably there is boilerplate code to get it to work on each page we want to display it.